src/events.js

```diff
@@ -11,55 +11,38 @@ const log = debug('watsonwork-weather-events');
 // event, the original annotated message and the user who sent it
 const callback = (evt, appId, info, annotation, token, cb) => {
 
-  // Retrieve the annotated message
+    // Retrieve the annotated message
```

Comment:
Not sure why the formatting changes here (we've been using spaces for indentation, is this a tab?)